Automattic/jetpack
# Remove blaze link on posts and pages

## Proposed changes:

* Remove 'Promote with Blaze' from page and post items on the Pages page and Posts page

### Other information:


## Jetpack product discussion

pbNhbs-9Bz-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Make sure Blaze is enabled on your site (`wp-admin/admin.php?page=jetpack#/traffic`)
![image](https://github.com/Automattic/jetpack/assets/65001528/31d1fee8-8755-4f88-be2f-d1b9715a20ce)
3. Go to `/wp-admin/edit.php` and hover over the a post. Make sure "Promote with Blaze" does not show up
![image](https://github.com/Automattic/jetpack/assets/65001528/06e0b5d7-32e8-40e7-99b1-1d8030170b94)
4. Go to `/wp-admin/edit.php?post_type=page` and make sure it doesn't show up there either
![image](https://github.com/Automattic/jetpack/assets/65001528/48b8ce54-9ec8-40ef-a3ac-5f3f209a93a9)
